client/packages/programs/src/JsonForms/components/ProgramEvent.tsx

```diff
@@ -100,29 +108,30 @@ const UIComponent = (props: ControlProps) => {
 
   const encounterId = useEncounter.utils.idFromUrl();
```

Comment:
For future work, we could check in the DocumentRegistry if the current document is an encounter so that we don't need to rely on the url here...